go.mod

```
@@ -1,20 +1,37 @@
 module github.com/minio/hperf
 
-go 1.22
+go 1.22.4
```

```suggestion
go 1.22
```
No need to pin exact version in `go.mod`. This will mostly just be annoying with no upsides.